packages/rollup/src/collect.ts

```diff
@@ -0,0 +1,17 @@
+import { sync } from 'glob';
+import { resolve } from 'path';
+import type { InputOptions } from 'rollup';
+
+export const collect = async (
+  filePatterns: readonly string[],
+  dirname = process.cwd()
+): Promise<readonly InputOptions[]> => {
+  const files = filePatterns.flatMap((pattern) => sync(pattern) as readonly string[]);
+
+  const configs = files.map(async (file) => {
+    const config = await import(resolve(dirname, file));
+    return config.default;
+  });
```

Comment:
Prefer `.then` to async/await:
```suggestion
   const configs = files.map((file) => import(resolve(dirname, file)).then((cfg) => cfg.default));
```